src/libs/actions/Report.js

```diff
@@ -708,6 +510,7 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
                 isLoadingMoreReportActions: false,
                 lastVisitedTimestamp: Date.now(),
                 lastReadSequenceNumber: getMaxSequenceNumber(reportID),
+                reportName: CONST.REPORT.DEFAULT_REPORT_NAME,
```

Comment:
This caused [a regression](https://github.com/Expensify/App/issues/12629). Whenever a report is opened the name is reset to the default, and it's only set back to the proper name, like `#admins`, when the response comes back.

Reply:
Thanks for letting me know and taking care of it!